packages/dx-chart-core/src/utils/scale.ts

```diff
@@ -246,7 +231,24 @@ const invertBandScaleBounds = (scale: ScaleObject, range: NumberArray): DomainBo
   ];
 };
 
+// tslint:disable-next-line: ban-types
+const makeScaleHelper = <T extends Function>(choices: [T, T]) => {
+  const func: any = (scale: ScaleObject, ...args: any[]) => {
+    const choosen = choices[Number(!!scale.bandwidth)];
```

Comment:
Maybe we should write the next sentence more cleary? I mean something like this
```js
const choosen = scale.bandwidth === 0 ? choices[0] : choices[1]
```

Reply:
`scale.bandwidth` is a function (if exists) - not a number. That would be

```js
const choosen = 'bandwidth' in scale ? choices[0] : choices[1]
```

The question is - will accessing `choices` variable twice be more clear?

Reply:
I think so.

@Krijovnick what do you think about this?

Reply:
That code is no more.

Reply:
The last variant is more clear.